hugegraph-core/src/main/java/com/baidu/hugegraph/util/ConfigUtil.java

```diff
@@ -50,24 +63,36 @@
     private static final String CHARSET = "UTF-8";
 
     public static void checkGremlinConfig(String conf) {
-        YamlConfiguration yamlConfig = new YamlConfiguration();
+        Parameters params = new Parameters();
         try {
-            yamlConfig.load(conf);
+
+            FileBasedConfigurationBuilder<FileBasedConfiguration> builder =
+                new FileBasedConfigurationBuilder(YAMLConfiguration.class)
+                    .configure(params.fileBased().setFileName(conf));
+            YAMLConfiguration config = (YAMLConfiguration) builder
+                                                           .getConfiguration();
+
+            List<HierarchicalConfiguration<ImmutableNode>> nodes =
+                                           config.childConfigurationsAt(
+                                           NODE_GRAPHS);
+            E.checkArgument(nodes == null || nodes.isEmpty()
+                            || nodes.size() == 1,
```

Comment:
return  if `nodes == null || nodes.isEmpty()`, then check `nodes.size() == 1`

Reply:
E.checkArgument(nodes == null || nodes.isEmpty())
E.checkArgument(nodes.size() == 1)
do i change it to this?